config/routes.rb

```diff
@@ -7,6 +7,24 @@
 
   get 'users/styleguide' => 'users#styleguide', as: :signup
 
-  resources :organization_campaigns, only: :show
-  resources :matches, only: :create
+  resources :memberships, only: [:index, :new, :create, :edit, :update, :delete]
+  resources :matches, only: [:new, :create, :delete]
+  resources :organizations, only: [:show] do
+    member do
+      get :admins
+      get :new_admin
+      get :switch_current_campaign
+      get :import_emails_form
+      post :import_emails_form_page2
+    end
+  end
+  resources :organization_campaigns, only: [:show,:create] do
+    member do
+      get :send_email_form
+      get :unmatched
+      get :unmatched_tags
+      get :matched
+      get :match_form
+    end
+  end
 end
```

Comment:
think this is fine because we are making an admin dashboard, but in the future should we consider trying to use as many RESTful routes as possible,  as it will make it easier if we decide later on to make the switch to an API.